omod/src/test/java/org/openmrs/module/idgen/web/search/LogEntrySearchHandlerTest.java

```diff
@@ -0,0 +1,92 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.search;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import org.junit.Assert;
+
+public class LogEntrySearchHandlerTest extends MainResourceControllerTest {
+    private IdentifierSourceService service;
+    public static String USER_UUID = "1010d442-e134-11de-babe-001e378eb67e";
+    public static String LOG_ENTRY_SOURCE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a901";
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSet("org/openmrs/module/idgen/include/TestData.xml");
+        this.service = Context.getService(IdentifierSourceService.class);
```

Comment:
When do you ever use the above service?

Reply:
In the [getAllCount()](https://github.com/cnagadya/openmrs-module-idgen/blob/995126def04e560fbfb5df62d97f3196b30866f1/omod/src/test/java/org/openmrs/module/idgen/web/controller/LogEntryControllerTest.java#L34) method. I've moved it though

Reply:
What do you mean by you have moved it?

Reply:
Think I didnt commit the changes in this file. They'll reflect in the new commit